robots/panda_description/README.md

```diff
@@ -0,0 +1,4 @@
+# Panda ign
+
+upstream: https://github.com/AndrejOrsula/panda_ign
```

Comment:
Is this the origin or is it https://github.com/frankaemika/franka_ros/tree/noetic-devel/franka_description ? Which one should we use?

Reply:
The description on your link is Apache 2.0, which is the license provided by @jcarpent, so this is probably the right one. I'll fix this.